log.go

```diff
@@ -40,7 +40,9 @@ func (s *LogNode) runLog([]byte) error {
 	case pipeline.StreamEdge:
 		for p, ok := s.ins[0].NextPoint(); ok; p, ok = s.ins[0].NextPoint() {
 			buf.Reset()
-			env.Encode(p)
+			if err := env.Encode(p); err != nil {
+				return err
```

Comment:
Returning here will stop the task execution. The err should simply be logged a la

`s.logger.Println("E!", err)`